Тесты по методам GetBooking и GetBookinbyId

Alexandrsuslakov opened the pull request.

Alexandrsuslakov committed:
Тесты по методам GetBooking и GetBookinbyId

TsakunovR commented on src/main/java/core/settings/ApiEndpoints.java:
```diff
@@ -0,0 +1,17 @@
+public enum ApiEndpoints {
+    PING("/ping"),
+    BOOKING("/booking"),
+    BOOKINGBYID("/booking/2");
```
Удали этот ендпоинт

TsakunovR commented on src/main/java/core/clients/APIClient.java:
```diff
@@ -0,0 +1,78 @@
+
+
+    }
+    public Response getBookingById(){
```
id должен быть указан параметром функции

TsakunovR commented on src/main/java/core/clients/APIClient.java:
```diff
@@ -0,0 +1,78 @@
+    public Response getBookingById(){
+        return getRequestSpec()
+                .when()
+                .get(ApiEndpoints.BOOKINGBYID.getPath())
```
Так будет некорректно написать. Теряется универсальность метода. А если нам понадобится найти бронирование с другим id? Вспоминай как передавали параметр в прошлом проекте.

Alexandrsuslakov committed:
1.Удалил эндпоинт c id из ApiEndpoints
2.Добавил передачу параметра bookingid в функцию  getBookingById
3.Сделал тест параметризированным.
